doc/releases/release-notes-2.5.rst

```diff
@@ -79,6 +79,9 @@ API Changes
   standard way to identify the control signal in nodes that support power
   control.
 
+* The :c:type:`fs_tile_t` objects require initialization, with call to the
+  :c:func:`fs_file_t_init`, after definition or before first use.
```

Comment:
Minor editorial suggestion:

~The~ :c:type:`fs_tile_t` objects ~require initialization, with call to the~ *must now be initialized by calling* :c:func:`fs_file_t_init` ~, after definition or~ before *their* first use.

Reply:
Suggestion followed.